### PR TITLE
add arcGrid support

### PR DESCRIPTION
**Pull request by thinking-aloud**
Hi we need ArcGrid for our project, so I want ahead and implemented it for geoserver-manager.

### CODE REVIEW

File: src/test/java/it/geosolutions/geoserver/rest/publisher/GeoserverRESTArcGridTest.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+import static org.junit.Assert.*;
+
+/**
+ * Testcase for publishing layers on geoserver.
+ * We need a running GeoServer to properly run the tests. 
+ * If such geoserver instance cannot be contacted, tests will be skipped.
+ *
+ * @author Carlo Cancellieri - carlo.cancellieri@geo-solutions.it
+ */
```
Comment (dromagnoli) [inline src/test/java/it/geosolutions/geoserver/rest/publisher/GeoserverRESTArcGridTest.java]: I would use the proper author here :)

File: src/test/java/it/geosolutions/geoserver/rest/publisher/GeoserverRESTArcGridTest.java [hunk 2]
```diff
@@ -0,0 +1,179 @@
+/*
+ *  GeoServer-Manager - Simple Manager Library for GeoServer
+ *  
+ *  Copyright (C) 2007,2011 GeoSolutions S.A.S.
+ *  http://www.geo-solutions.it
```
Comment (dromagnoli) [inline src/test/java/it/geosolutions/geoserver/rest/publisher/GeoserverRESTArcGridTest.java]: I would put 2016 here instead.

**Comment by dromagnoli** [comment]
Hi @thinking-aloud.
Thanks for the contribution.
I have added a few trivial comments to the pull-request. 

I would also consider creating an "issue" for this work and associate the pull to the issue so that we can add a note in future release reports.
https://github.com/geosolutions-it/geoserver-manager/issues/new

File: src/main/java/it/geosolutions/geoserver/rest/GeoServerRESTPublisher.java [hunk 3]
```diff
@@ -1520,6 +1522,215 @@ private boolean publishExternalCoverage(String workspace, String coveragestore,
...
+                (coverageName != null) ? new NameValuePair[] { new NameValuePair("coverageName",
+                        coverageName) } : (NameValuePair[]) null);
+    }
+
+    /**
+     * Same as {@link #publishArcGrid(String, String, String, File, String, ProjectionPolicy, String, double[])} but without the last parameter
+     * (bbox). Kept here for backwards compatibility.
+     *
+     * @deprecated use the former method with bbox set to null.
+     */
```
Comment (dromagnoli) [inline src/main/java/it/geosolutions/geoserver/rest/GeoServerRESTPublisher.java]: Being a new feature, I would avoid adding a deprecated method. 
You can directly rely on the method with the bbox param and remove this one.

**Comment by thinking-aloud** [comment]
#173

**Comment by thinking-aloud** [comment]
thx @dromagnoli I updated the year and autor.

**Pull request merged**